src/index.json

```diff
@@ -2181,6 +2181,52 @@
                 },
                 "sha256Digest": "33e87daa9ef75664e9c49f619985a0869ba6606c2a76cd5c6329f309f0e828b1"
             }
+        ],
+		"vm-repair": [
+            {
+                "downloadUrl": "https://azurecomputeaidrepair.blob.core.windows.net/prod/vm_repair-0.0.1-py2.py3-none-any.whl",
+                "filename": "vm_repair-0.0.1-py2.py3-none-any.whl",
```

Comment:
The initial version should be 0.1.0 instead of 0.0.1

Reply:
changed to 0.1.0